madlibs/madlibs_custom.py

```diff
@@ -0,0 +1,126 @@
+'''
+This program is an example solution of what your mad libs game could look like! The code is simple implementation of the game. Are there any ways you can improve it? :)
+'''
+
+# Colour class for fancy text
+# Syntax: bcolours.BLUE or GREEN + STRING + bcolours.END
+class bcolours:
+    BLUE = '\033[94m'
+    GREEN = '\033[92m'
+    END = '\033[0m'
+
+print("\nWelcome to PETCS Mad Libs!\n")
+print('''Info --------------------->
+Copy/paste a custom 1 paragraph story that follows the legend!
+If you want to input a .txt file, make sure the .txt file is in the same directory (folder) as the python script!
+
+Recommended story length is 100 - 500 words, but I won't stop you in testing the limits :) 
+
+Legend for user input:
+ADJ - Ask for an adjective
+NOUN - Ask for a noun
+VERBP - Ask for a plural verb
+VERBS - Ask for a singular verb
+VERBD - Ask for a past tense verb
+ADVERB - Ask for a adverb
+* Legend is case sensitive!
+''')
+  
+keepPlaying = True
+repeat = False
+
+# Template story
+story_temp = []
+copy = []
+
+# Allows the user to upload a custom story
+def upload_story():
+  if repeat:
+    return copy[:]
+  else:
+    story = ""
+    while True:
+      choice = input("Upload .txt file or manually (txt/man): ").lower()
+      # This code assumes you put in the right .txt name.
+      # However, if you put the wrong name, it will throw an error. How can you work around that? 
+      # Hint: Try Except
+      if choice == "txt":
+        # Although we didn't teach open(), it is a built-in function for opening files
+        open_story = open(input("Enter file name: "), "r")
+        story = open_story.read()
+        break
+      elif choice == "man":
+        story = input("Type/paste your story here: ")
```

Comment:
Should probably make it clear that while typing the story then shouldn't use the `enter` key?

Reply:
Yeah good point. I'll specify that!